client/app/bundles/course/user-invitations/operations.ts

```diff
@@ -132,7 +132,7 @@ export function toggleRegistrationCode(shouldEnable: boolean): Operation {
 }
 
 const splitNameAndEmailRegex =
-  /^(?:"\s*([^"]+?)\s*"|\s*([^"<]+?))?\s*<\s*([^>\s]+?)\s*>$|^\s*([^"<\s]+@[^\s,;<>]+)\s*$/;
+  /^(?:\s*"\s*([^"]+?)\s*"|\s*([^"<]+?))?\s*<\s*([^>\s]+?)\s*>$|^\s*([^"<\s]+@[^\s,;<>]+)\s*$/;
```

Comment:
`/^\s*(?:(?:(?:"(?=.*")(.*)"|(.*?))\s*(?=<)<\s*(?=.+>)(\S+)\s*>)|(.+?))\s*$/;`  
Help double check this works.

The current one doesn't handle trailing spaces.

Reply:
done